xa/java/department-spring-jpa/src/main/java/com/oracle/mtm/sample/XADataSourceConfig.java

```diff
@@ -175,7 +175,7 @@ public EntityManagerFactory createEntityManagerFactory() throws SQLException {
         Properties properties = new Properties();
         properties.setProperty( "jakarta.persistence.transactionType", "RESOURCE_LOCAL"); // change this to resource_local
         properties.put("hibernate.show_sql", "true");
-        properties.put("hibernate.dialect", "org.hibernate.dialect.Oracle12cDialect");
```

Comment:
We don't need createEntityManagerFactory() method anymore right? if not used, let's remove